### DISCUSSION TITLE
How to create Python and Java portable versions

### DISCUSSION THREAD

**Discussion by LrkDev** [leokhoa/laragon#493] [Q&A]
Hello @leokhoa,

It would be nice if you could (when you've got time) publicize the method you used to create portable Python and jdk like the one you published as assets in the link below.
https://github.com/leokhoa/laragon/releases/tag/portable

This way community could help make newer versions packages.

This solution #411 is nice but don't come with pip

Best regards.

**Comment by lyingapril** [comment]
I also want to know how to add java. the link is expired. https://forum.laragon.org/topic/652/tutorial-how-to-add-java-9-to-laragon-the-portable-way

**Comment by lyingapril** [comment]
ohh, I found something here, https://laragon.org/docs/quick-add.html

**Reply by dokimkhanh** [reply]
> ohh, I found something here, https://laragon.org/docs/quick-add.html

Hey, i click "ALL" but not found Java. How u can install Java with Laragon ?
